crates/sui-protocol-config/src/lib.rs

```diff
@@ -804,6 +839,16 @@ impl ProtocolConfig {
                 event_tag_size_derivation_cost_per_byte: Some(1_000),
                 // Emitting an event is cheap since its a vector push
                 event_emit_cost_per_byte: Some(1_000),
+                // Crypto
+                // ed25519
+                ed25519_ed25519_verify_cost_base: Some(52),
+                ed25519_ed25519_verify_msg_cost_per_byte: Some(0),
+                // hash::blake2b256
+                hash_blake2b256_cost_base: Some(52),
+                hash_blake2b256_data_cost_per_byte: Some(0),
+                // hash::keccak256
+                hash_keccak256_cost_base: Some(52),
```

Comment:
lease remember that the cost numbers are just placeholders right now. We’ll finetune.